app/backend/src/couchers/servicers/search.py

```diff
@@ -42,10 +42,13 @@ def _join_with_space(coalesces):
     return out
 
 
-def _build_tsv(A, B=[], C=[], D=[]):
+def _build_tsv(A, B=None, C=None, D=None):
     """
     Given lists for A, B, C, and D, builds a tsvector from them.
     """
+    B = B if B else []
```

Comment:
Why not `B=B or []` here too?

Reply:
Oops, missed one from what I had previously.